### DISCUSSION TITLE
Sign In with Ethereum (SIWE) ERROR

### DISCUSSION THREAD

**Discussion by dadsec-dev** [rainbow-me/rainbowkit#1778] [Q&A]
Hello, 

I am trying to implement sign in with ethereum using the siwe provider in order to authenticate users and get sessions but i've been getting this error

![twitter_cuygFpCY](https://github.com/rainbow-me/rainbowkit/assets/77055488/4577f817-b6a7-4b5c-b953-59e2a5676161)

help please ....

**Comment by 0xmkh** [comment]
Hey @dadsec-dev. We're happy to help! Would you be able to give us a small reproducible example ? Also what version of RainbowKit and Next auth are you using ?

Would also like to know the wallet you're using when testing this as well 🙏

Meanwhile you can also check our example [here](https://github.com/rainbow-me/rainbowkit/tree/main/examples/with-next-siwe-next-auth).
